### ISSUE TITLE
Link to "latest Java version"

### ISSUE THREAD

**Issue by stevenschlansker** [shyiko/jabba#6]
Right now, if you set up any references to JVMs, you'll put the patch number in the location.  For example,

``` xml
  <!-- JDK toolchains -->
  <toolchain>
    <type>jdk</type>
    <provides>
      <version>1.8</version>
      <vendor>oracle</vendor>
    </provides>
    <configuration>
      <jdkHome>/Users/steven/.jabba/jdk/1.8.77/Contents/Home</jdkHome>
    </configuration>
  </toolchain>
```

It would be nice if Jabba could maintain a `1.8` jdk which is a symlink to the latest installed `1.8.x`

**Comment by shyiko** [comment]
@stevenschlansker This is an excellent idea! Will do. Thank you.

**Comment by liaoya** [comment]
It will be better if there is a link for current JDK. You can take a look at [junction](https://technet.microsoft.com/en-us/sysinternals/bb896768) or [ln.ps1](https://github.com/lukesampson/psutils/blob/master/ln.ps1) for link support on Windows

**Comment by shyiko** [comment]
For the record, you can use `jabba link` to achieve something similar (e.g. `jabba link system@1.8 $(jabba which)`). It's not the same as the original feature request though.

**Comment by shyiko** [comment]
@stevenschlansker @liaoya I'm happy to announce that this issue is effectively resolved (both your use-cases are implemented in 0.9.0) :tada: 

(see [0.9.0 release notes](https://github.com/shyiko/jabba/blob/master/CHANGELOG.md#090---2017-09-19) for details)

**Comment by shyiko** [comment]
(to link existing JDKs execute `jabba link`; links are automatically updated on each `jabba install`/`jabba uninstall` & `jabba alias default <semver>`)
